app/src/bcsc-theme/features/verify/VerifyIdentityStack.tsx

```diff
@@ -90,6 +95,11 @@ const VerifyIdentityStack = () => {
       <Stack.Screen name={BCSCScreens.TransferAccountInstructions} component={TransferInstructionsScreen} />
       <Stack.Screen name={BCSCScreens.TransferAccountQRScan} component={TransferQRScannerScreen} />
       <Stack.Screen name={BCSCScreens.IdentitySelection} component={IdentitySelectionScreen} />
+      <Stack.Screen name={BCSCScreens.EditNickname} component={EditNicknameScreen} />
+      <Stack.Screen name={BCSCScreens.ForgetAllPairings} component={ForgetAllPairingsScreen} />
+      <Stack.Screen name={BCSCScreens.HelpCentre} component={HelpCentreScreen} />
+      <Stack.Screen name={BCSCScreens.PrivacyPolicy} component={PrivacyPolicyScreen} />
+      <Stack.Screen name={BCSCScreens.ContactUs} component={ContactUsScreen} />
```

Comment:
HelpCentre and ContactUs should have their titles included. Can copy them from the other stack